components/dashboards-web-component/src/designer/DashboardDesigner.jsx

```diff
@@ -269,7 +274,9 @@ export default class DashboardDesigner extends Component {
                 });
             })
             .catch((err) => {
-                //TODO Need to use proper notification library to show the error
+                if (err.response.status === 401) {
+                    this.setState({hasPermission: false});
+                }
```

Comment:
what happens if `err.response.status` is 400 or any other value?

Reply:
We decided to handle error pages as a separate improvement.